salt/docker_apps/files/kingfisher_process.yaml

```
@@ -11,6 +11,10 @@ x-filesystem: &filesystem
   volumes:
     # The right-hand path must match the settings.KINGFISHER_COLLECT_FILES_STORE default value in kingfisher-process.
     - {{ pillar.kingfisher_collect.env.FILES_STORE }}:/data
+    # For processing local loads.
+    {% for user, authorized_keys in pillar.users.items() %}
+    - /home/{{user}}/local-load:/home/{{user}}/local-load
+    {% endfor %}
```

Change to a Jinja comment, otherwise this comment will appear on the registry server, without anything beneath it.

Change pillar.users to optional, since the registry server doesn't have this.

```suggestion
    {# To run the load command. #}
    {%- for user in pillar.users|default({}) %}
    - /home/{{user}}/local-load:/home/{{user}}/local-load
    {%- endfor %}
```